share/rocmcmakebuildtools/cmake/ROCMCreatePackage.cmake

```diff
@@ -465,17 +467,20 @@ macro(rocm_setup_license HEADER_ONLY)
 
     if(CPACK_RESOURCE_FILE_LICENSE)
         if(ENABLE_ASAN_PACKAGING)
+            message(STATUS "rocm-cmake: set install license file to ${_rocm_cpack_package_name}-asan.")
             install(
                 FILES ${CPACK_RESOURCE_FILE_LICENSE}
                 DESTINATION share/doc/${_rocm_cpack_package_name}-asan
             )
         elseif((ROCM_USE_DEV_COMPONENT AND ${HEADER_ONLY}) OR NOT BUILD_SHARED_LIBS)
+            message(STATUS "rocm-cmake: set install license file to ${HDRONLY_LICENSE_INSTALLDIR}.")
             install(
                 FILES ${CPACK_RESOURCE_FILE_LICENSE}
-                DESTINATION share/doc/${_rocm_cpack_package_name}
+                DESTINATION share/doc/${HDRONLY_LICENSE_INSTALLDIR}
```

Comment:
Just to be safe, can we keep the old behaviour when `HDRONLY_LICENSE_INSTALLDIR` is unset?